src/github.ts

```diff
@@ -183,3 +194,41 @@ export async function mergePR(client: Octokit, db: Database, pendingPR: PendingP
     return false;
   }
 }
+
+/**
+ * Returns list of approvers from the OWNERS file at the base of the repo.
+ * @param {Octokit} client the GitHub client to use that is authenticated as the GitHub installation.
+ * @param {string} organization the GitHub organization of the repository to check.
+ * @param {string} repoName the GitHub repository name of the repository to check.
+ * @param {string} branchName the GitHub branch name to check.
+ * @param {Array<string>} prAuthor the author of the pull request.
+ * @return {Promise<boolean>} a Promise that resolves to whether the given author is also an owner.
+ */
+export async function getOwners(
+  client: Octokit,
+  organization: string,
+  repoName: string,
+  branchName: string,
+): Promise<string[]> {
+  const ownersFileObject = await client.repos.getContent({
+    owner: organization,
+    repo: repoName,
+    path: "OWNERS",
+    ref: branchName,
+    mediaType: { format: "application/vnd.github+json" },
+  });
```

Comment:
Is it ok we don't add `catch`?

Reply:
That's a good point, especially since GitHub can flake sometimes. I'll see about catching it.

Reply:
If it's only run in the syncer, then it's okay because if anything throws an exception, a message is logged and it tries again on the next poll:
https://github.com/stolostron/magic-mirror/blob/9c521466939cd3cb916a7ebbba0008f8751c0d78/src/syncer.ts#L598-L602

The one `web.ts` is a bit trickier because nothing retries those so maybe you'd rather just default to no assignees if this failed.

Reply:
Sounds good--I'm adjusting the `web.ts` to catch it and left the `syncer.ts` to catch the resulting error.